include/os/freebsd/spl/sys/isa_defs.h

```diff
@@ -41,59 +41,6 @@
  *	The natural byte order of the processor.  A pointer to an int points
  *	to the least/most significant byte of that int.
  *
- * _STACK_GROWS_UPWARD / _STACK_GROWS_DOWNWARD:
```

Comment:
I have wanted to revise this in favor of either the Linux approach or FreeBSD approach for a while now, but have not gotten to it. This will probably need revision yet again when the day comes that we officially support Illumos/OpenSolaris in the OpenZFS tree again, but I am fine with this for the time being. I honestly have no preference on how we do this in the current tree.

Reply:
Moving in the direction of the Linux version makes sense to me. This was just a first cut at removing cruft that matters to a whole OS, but isn't used in ZFS at all.